floyd/model/experiment_config.py

```diff
@@ -6,7 +6,7 @@
 class ExperimentConfigSchema(Schema):
 
     name = fields.Str()
-    family_id = fields.Str()
+    family_id = fields.Str(allow_none=True)
```

Comment:
Do you know what workflow triggers this? I was under the impression that this should not happen.

Reply:
The workflow is that in order to allow the `floyd init` command to succeed before the project exists on floydhub, we need to save the config without the family_id, then be able to get it later on. So before the project is created, family_id needs to be nil. Is there a cleaner way to do this that you can think of?

Reply:
Ha, I see. We can also use the same approach that's used by login, i.e. wait for user to come back from web UI and press enter to confirm the project creation. That way you will be 100% sure the project is created. We can even pull in the cli while user is creating the project and exit automatically.